### PR TITLE
Work vs Collection Help Text

### LINKED ISSUE #604: Create the Help Text for Collection vs Work
We need help text to help the user decide if they want to create a work or a collection.

What should the wording for that be?

### PR DESCRIPTION
**Pull request by mtribone**
Adjusts text for new work and adds work vs collection text to new collection to be consistent.

Fixes #604 and #599

### CODE REVIEW

**Comment by mtribone** [comment]
screen shot 2017-04-24 at 2 11 26 pm

**Comment by mtribone** [comment]
screen shot 2017-04-24 at 2 10 54 pm

File: app/views/collections/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<h1>Create New Collection</h1>
+<p class="instructions"><%= t("sufia.batch_uploads.files.concepts") %></p>
```
Comment (awead) [inline app/views/collections/new.html.erb]: @mtribone shouldn't this have it's own entry? The text for `sufia.batch_uploads.files.concepts` doesn't seem appropriate for collections.

**Review by awead — CHANGES_REQUESTED** [review]

File: app/views/collections/new.html.erb (hunk 1 above)
Comment (mtribone) [reply app/views/collections/new.html.erb]: The text is exactly the same. If I give it its own entry we'll need to change it twice. Perhaps a better thing would be to pull it out of batch_uploads and put it in a more generic place?

Comment (awead) [reply app/views/collections/new.html.erb]: @mtribone Maybe I'm misunderstanding, but here's what I figure that page will show:

# Create New Collection
A work consists of one or more files that work together as a functional unit [...]

I thought the idea was to explain what collections are. Why are we describing what a work is if we're creating collections?

Comment (awead) [reply app/views/collections/new.html.erb]: duh. sorry, screenshot is above. So, yeah, all I'm saying is it looks odd to be talking about works when you're on a collections creation page.

Comment (mtribone) [reply app/views/collections/new.html.erb]: Yeah I can see that point. I was asked today to add the same help text for consistency.

**Comment by mtribone** [comment]
New collection help text


screen shot 2017-04-24 at 4 10 01 pm

**Comment by mtribone** [comment]
screen shot 2017-04-25 at 9 42 38 am

**Review by awead — APPROVED** [review]

**Pull request merged**